docs/tutorials/text-classification-tutorial.mdx

```diff
@@ -0,0 +1,295 @@
+
+- Login:
+
+`mysql -u user -p`
+
+- Create your database:
+
+`CREATE DATABASE nlp_mindsdb;`
+
+- Create a table:
+
+```sql
+CREATE TABLE tweets (
+  id INTEGER,
+  hate_speech INTEGER,
+  offensive INTEGER,
+  neither INTEGER,
+  tweet VARCHAR(255),
+  PRIMARY KEY (id)
+);
+```
+
+---
+
+Query OK, 0 rows affected (0.03 sec)
+
+mysql> SHOW TABLES;
++-----------------------+
+| Tables_in_nlp_mindsdb |
++-----------------------+
+| tweets                |
++-----------------------+
+1 row in set (0.00 sec)
+
+---
+
+- Next grant the user access to the new database table:
+
+'GRANT ALL PRIVILEGES ON `nlp_mindsdb.tweets` TO user@localhost;'
+
+***output should read:***
+`Query OK, 0 rows affected (0.01 sec)`
+
+---
+
+## Understanding the Data
+
+[Hate-speech-CNERG/bert-base-uncased-hatexplain](https://huggingface.co/Hate-speech-CNERG/bert-base-uncased-hatexplain?text=I+like+you.+I+love+you) is The Hugging Face transformer model you will be using for this exercise. The model is used for classifying a text as Hatespeech, Offensive, or Normal. The model is trained using data from Gab and Twitter and Human Rationales were included as part of the training data to boost the performance. All of the datasets and models for this transformer are available [here](https://github.com/hate-alert/HateXplain).
+
+This next section will show you how to import this [dataset file](https://raw.githubusercontent.com/hate-alert/HateXplain/master/Data/dataset.json), which was converted into a .cvs for this tutorial, into your local MySQL table you created above:
+
+- Load the data from your dataset into the `tweets` table:
+
+```sql
+mysql> LOAD DATA LOCAL INFILE '/Users/alissa/Downloads/tweets.csv' INTO TABLE tweets FIELDS TERMINATED BY `,`;
+```
+
+```
+Query OK, 24784 rows affected, 17300 warnings (0.26 sec)
+Records: 26952  Deleted: 0  Skipped: 2168  Warnings: 17300
+```
+
+---
+
+- Go ahead and confirm your import:
+
+`SELECT * FROM tweets LIMIT `0`
+
+```sql
+------------------------------------------------------------+
+    -> | id | hate_speech | offensive | neither | tweet                                                                                                                                         |
+    -> +----+-------------+-----------+---------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+    -> |  0 |           0 |         0 |       3 | !!! RT @mayasolovely: As a woman you shouldn't complain about cleaning up your hou |. &amp; as a man you should always take the trash out...
+    '> |  1 |           0 |         3 |       0 | !!!!! RT @mleew17: boy dats cold...tyga dwn bad for cuffin dat hoe in the 1st plac                                                        |
+    '> |  2 |           0 |         3 |       0 | !!!!!!! RT @UrKindOfBrand Dawg!!!! RT @80sbaby4life: You ever fuck a bitch and she                     |confused as shit
```

Comment:
Please remove all offensive/vulgar words and examples from this tutorial. MindsDB documentation shouldn't contain any negative language or hate speech.

Reply:
ahhh I thought I did! I'm sorry I missed those @martyna-mindsdb  - ok, on it now